spateo/plotting/static/polarity.py

```diff
@@ -7,7 +7,7 @@
 from anndata import AnnData
 
 
-def digi_exp_plot(
+def polarity(
```

Comment:
I'll work on adding confidence interval.

Reply:
wonderful

Reply:
please also make sure you will add the typing, documentations, some log and comments to these plotting functions. Thanks

Reply:
also, may think creatively what other analyses we can do? how about adapting the `glm_deg` code from dynamo to detect genes that change as a function of the layer or columns? see: https://github.com/aristoteleo/dynamo-release/blob/6e04f0556b8d2faae9269766c8e54eb1bc617048/dynamo/tools/markers.py#L626